### ISSUE TITLE
FAB background color is not the same as the others button.

### ISSUE THREAD

**Issue by mai1202** [material-components/material-web#5340]
### What is affected?

Theming

### Description

I used 
```
import { argbFromHex, themeFromSourceColor, applyTheme } from "@material/material-color-utilities";

const theme = themeFromSourceColor(argbFromHex('#00a0ff'));
console.log(theme);
applyTheme(theme, { target: document.body });
```
to apply a theme to my app. 

However, it is using the default color unlike the others.
![image](https://github.com/material-components/material-web/assets/63698132/67cac369-6c92-486a-8f5d-7e5d7384fb0d)

### Reproduction

It should use a themeing color.

### Workaround

I have not found a workdground.

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

v1.1.1

### Browser/OS/Node environment

Google Chrome 120.0.6099.111
Windows 11
v18.16.1
9.5.1

**Comment by asyncLiz** [comment]
This is related to https://github.com/material-components/material-web/issues/5099#issuecomment-1808786172

tl;dr, the `@material/material-color-utilities` package is out of date right now. We recommend using https://material-web.dev/ and its color palette copy button for generating themes.

The comment above also has links to our catalog code, which adds the features needed to the color utilities for runtime dynamic color. You're welcome to copy!
